physics/lsm_ruc.F90

```diff
@@ -365,18 +367,20 @@ subroutine lsm_ruc_run                                            & ! inputs
       implicit none
 
 !  ---  constant parameters:
-      real(kind=kind_phys), parameter :: rhoh2o  = 1000.0
-      real(kind=kind_phys), parameter :: stbolt  = 5.670400e-8
+      real(kind_phys), parameter :: rhoh2o  = 1000.0
```

Comment:
This should be passed in from the host:
`[rhoh20]`
 `standard_name = fresh_liquid_water_density_at_0c`
 `long_name = density of liquid water`
 `units = kg m-3`
 `dimensions = ()`
 `type = real`
 `kind = kind_phys`

Reply:
A change is done to use the water density from the host.